packages/terra-worklist-data-grid/src/subcomponents/ColumnHeader.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
 import WorklistDataGridPropTypes from '../proptypes/WorklistDataGridPropTypes';
 import ColumnHeaderCell from './ColumnHeaderCell';
 
 const propTypes = {
   /**
-   * Data for columns. By default, columns will be presented in the order given.
+   * Data for columns.
+   * Columns will be presented in the order given.
    */
-  columns: PropTypes.arrayOf(WorklistDataGridPropTypes.columnShape).isRequired,
+  columns: PropTypes.arrayOf(WorklistDataGridPropTypes.columnShape),
```

Comment:
Why would the columns array not be required?  You can't have a column header without columns.

Reply:
Most of the changes in this file were left over from our initial changes and have been reverted: [1a46886](https://github.com/cerner/terra-framework/pull/1710/commits/1a4688675452581e97d235eee07f053457a24860)